pytorch_toolkit/object_detection/oteod/oteod/args_conversion.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from ote import MODEL_TEMPLATE_FILENAME
+
+
+def convert_ote_to_oteod_train_args(template_folder, args):
+    update_config_map = {
+        'train_ann_files': 'data.train.dataset.ann_file',
+        'train_img_roots': 'data.train.dataset.img_prefix',
+        'val_ann_files': 'data.val.ann_file',
+        'val_img_roots': 'data.val.img_prefix',
+        'resume_from': 'resume_from',
+        'load_weights': 'load_from',
+        'save_checkpoints_to': 'work_dir',
+        'batch_size': 'data.samples_per_gpu',
+        'base_learning_rate': 'optimizer.lr',
+        'epochs': 'total_epochs',
+    }
+
+    update_config = {v: args[k] for k, v in update_config_map.items()}
```

Comment:
It is just my opinion but maybe ```updated_config``` instead of ```update_config``` will reflect the essence of this value more clearly, because the common config has been modified in the mmdetection style